domains/zeaxey.json

```diff
@@ -8,5 +8,6 @@
     },
     "record": {
         "CNAME": "pixel556.github.io"
+        "TXT": "dh=03271c5828f57362f1d12969313966fb88b6f83e"
```

Comment:
not the way to do discord txt verification, you need to put it in _discord.zeaxey.json